steps/property-inspection/garden.tsx

```diff
@@ -128,7 +128,8 @@ const step = {
           renderWhen(stepValues: {
             "garden-type"?: ComponentValue<ProcessDatabaseSchema, "property">;
           }): boolean {
-            return stepValues["garden-type"] === ("private" || "not sure");
+            return stepValues["garden-type"] === ("private") ||
+            stepValues["garden-type"] === ("not sure");
```

Comment:
Yup. That was a mistake.

You'll need to apply formatting here again. Run `npm run format`.

Reply:
I think this is now fixed. Could you please have another look?